### PR TITLE
[IOTDB-837] ArrayIndexOutOfBoundsException if the measurementId size is not consistent with the value size

### PR DESCRIPTION
**Pull request by wyh-ztf**
ArrayIndexOutOfBoundsException if the measurementId size is not consistent with the value size

### CODE REVIEW

**Review by neuyilan — COMMENTED** [review]
Please make the title clearer.

File: server/src/main/java/org/apache/iotdb/db/qp/strategy/PhysicalGenerator.java [hunk 1]
```diff
@@ -176,6 +176,13 @@ public PhysicalPlan transformToPhysicalPlan(Operator operator) throws QueryProce
               "For Insert command, cannot specified more than one seriesPath: " + paths);
         }
```
Comment (qiaojialin) [inline server/src/main/java/org/apache/iotdb/db/qp/strategy/PhysicalGenerator.java:177]: This check could also be removed. I do not know how to trigger this. The sql parser should have already checked.

File: server/src/main/java/org/apache/iotdb/db/qp/strategy/PhysicalGenerator.java [hunk 2]
```diff
@@ -176,6 +176,13 @@ public PhysicalPlan transformToPhysicalPlan(Operator operator) throws QueryProce
...
 
+        if (insert.getMeasurementList().length != insert.getValueList().length) {
+          throw new ArrayIndexOutOfBoundsException(
```
Comment (qiaojialin) [inline server/src/main/java/org/apache/iotdb/db/qp/strategy/PhysicalGenerator.java:180]: You could throw an SQLParserException.

**Review by qiaojialin — COMMENTED** [review]
Good job!

File: server/src/main/java/org/apache/iotdb/db/qp/strategy/PhysicalGenerator.java (hunk 2 above)
Comment (wyh-ztf) [reply server/src/main/java/org/apache/iotdb/db/qp/strategy/PhysicalGenerator.java:180]: Thank you for your opinion. I will revise it

**Review by qiaojialin — COMMENTED** [review]
I forget one thing. For each bug fix, please add a test to verify it

**Review by neuyilan — APPROVED** [review]

**Comment by wyh-ztf** [comment]
> I forget one thing. For each bug fix, please add a test to verify it

ok,i will add a test to verify it

**Review by qiaojialin — APPROVED** [review]

**Comment by qiaojialin** [comment]
Hi, we usually use square brackets [] around the issue number in the title

**Comment by wyh-ztf** [comment]
> Hi, we usually use square brackets [] around the issue number in the title

OK, I'll pay attention next time. Thanks for your help

**Pull request merged**